packages/core/helper-plugin/lib/src/components/ConfirmDialog/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Can you add some tests and improve the story so it show the possible configurations?